src/content/reference/okteto-manifest.mdx

```diff
@@ -96,6 +96,8 @@ Follow this document for a list of [environment variables](reference/okteto-cli.
 
 Use `okteto deploy --dependencies` to force the redeployment of your dependencies.
 
+### dependencies - extended notation (object, optional)
```

Comment:
Let's remove this line. That way everything is index under `dependencies`

Reply:
Is there a good way we can communicate that this property has a simple and an extended notation then? I like the idea of the formatting being consistent but need a way as a consumer to know that there are two explicit versions. (Sidenote, is there a YAML schema for the Okteto file somewhere?)

Reply:
We don't have a YAML schema for the Okteto Manifest yet :disappointed_relieved:
Are you familiar with any documentation where they document different field types?
I have been looking for inspiration, but didn't find anything great